onboard/gps/src/__main__.py

```diff
@@ -0,0 +1,162 @@
+import serial
+import asyncio
+import json
+import numpy as np
+from os import getenv
+from rover_common.aiohelper import run_coroutines
+from rover_common import aiolcm
+from rover_msgs import GPS, RTCM
+
+
+class GPS_Manager():
+
+    def __init__(self):
+
```

Comment:
NIT: remove? I noticed you're consistent with this. Is this needed for the linter? If this is considered good style, then go ahead and leave it.

Reply:
Again, I was talking about the whitespace which we discussed earlier is for the linter.